src/extractors/exported.js

```diff
@@ -124,18 +135,23 @@ export default function walkExported(
             );
             bindingPath = tmp.ast;
             specData = tmp.data;
-          } else if (exportKind === 'value') {
```

Comment:
are these kinds safe to skip or is it only for Ogma case? Is there a test for that?